public/js/test/mochitest/examples/README.md

```diff
@@ -0,0 +1,5 @@
+### Test Examples
+
+##### Pages
+* **doc_script-switching-01** - includes two scripts that reference each other. The second function has a debugger.
+* **doc-scripts** - includes three sources, a long source and two sources that reference each other.
```

Comment:
Also can you rename the new test file with the `doc-` prefix instead of `doc_`? Trying to avoid mixing underscores and dashes (but `browser_` is required by mochitest)